homeassistant/components/ipp/coordinator.py

```diff
@@ -18,7 +20,14 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-class IPPDataUpdateCoordinator(DataUpdateCoordinator[IPPPrinter]):
+class IPPDataUpdateCoordinatorType(TypedDict):
```

Comment:
I am never sure when to use dataclass and when to use TypedDict but personally I like dataclass more because it looks better, not a blocker, just saying :)

Reply:
I got this from the `fritz` integration and there from another integration and there ... yeah, i just stole it from other integrations :grin: